### PR TITLE
[FORM | TEXTAREA FORMATTING] FRONTEND / PDF BACKEND

### PR DESCRIPTION
**Pull request by rusllonrails**
[TRELLO STORY](https://trello.com/c/3OCMPeXD/596-qae17imp-as-an-applicant-i-would-like-to-be-able-to-add-formating-to-my-application)

### CODE REVIEW

File: app/pdf_generators/qae_pdf_forms/custom_questions/textarea.rb [hunk 1]
```diff
@@ -0,0 +1,321 @@
...
+  end
+
+  def wysywyg_handle_ol_tag(key, i)
+    li_style = styles_picker(@styles)
+    print_pdf(@string.join(""), li_style)
+
+    @ns_history << @counter
+    @counter = 0
+    @string = []
+    key = "<ol>"
```
Comment (dreamfall) [inline app/pdf_generators/qae_pdf_forms/custom_questions/textarea.rb]: You're reassigning a variable that was not used in the method before. Does this method need to accept `key` as an argument?

File: app/pdf_generators/qae_pdf_forms/custom_questions/textarea.rb [hunk 2]
```diff
@@ -0,0 +1,321 @@
...
+
+      else
+        @string << i
+      end
+    end
+
+    @string
+  end
+
+  def wysywyg_handle_list_ending_tag(key, i)
```
Comment (dreamfall) [inline app/pdf_generators/qae_pdf_forms/custom_questions/textarea.rb]: Neither `key`, nor `i` are not used here, can you please remove?

File: app/pdf_generators/qae_pdf_forms/custom_questions/textarea.rb [hunk 3]
```diff
@@ -0,0 +1,321 @@
...
+      print_pdf(@string.join(""), li_style)
+
+      if @li_counter > 1
+        (@li_counter - 1).times do
+          @styles.pop
+        end
+
+        @li_counter = 0
+      else
+        @li_counter = 0
```
Comment (dreamfall) [inline app/pdf_generators/qae_pdf_forms/custom_questions/textarea.rb]: You can move `@li_counter = 0` out of the `if` block.

**Comment by rusllonrails** [comment]
@dreamfall thanks, gonna fix :+1:

**Pull request merged**